### PR TITLE
feat: Added Slack link unfurling

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Unfurling is cool so we should have it! This _should_ respond to links in slack that match the instance domain.

## Changes

* Adds a new endpoint (ee) for handling events incoming from Slack
* Validates them using the signature env
* Finds the related asset and IF shared then generates an image to respond with

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

* Good news - unit tests!
* Bad news - can't test locally as they have to send us a webhook...

## TODO

- [ ] Maybe not always generate an asset... Maybe check if we have one already created and use it.

### CODE REVIEW

File: frontend/src/lib/components/Subscriptions/views/EditSubscription.tsx [hunk 1]
```diff
@@ -164,7 +164,7 @@ export function EditSubscription({
                         )}
 
                         <Field name={'title'} label={'Name'}>
-                            <LemonInput placeholder="e.g. Weekly team report" disabled={emailDisabled} />
+                            <LemonInput placeholder="e.g. Weekly team report" />
```
Comment (benjackwhite) [inline frontend/src/lib/components/Subscriptions/views/EditSubscription.tsx:167]: Random thing I found - this should never be disabled

**Review by pauldambra — APPROVED** [review]
Looks good...

File: ee/api/test/test_integration.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class TestIntegration(APILicensedTest):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        set_instance_setting("SLACK_APP_SIGNING_SECRET", "not-so-secret")
+
+    def _headers_for_paylod(self, payload: Any):
```
Comment (kappa90) [inline ee/api/test/test_integration.py:18]: typo :)

**Review by kappa90 — APPROVED** [review]

File: ee/api/test/test_integration.py (hunk 2 above)
Comment (benjackwhite) [reply ee/api/test/test_integration.py:18]: No it isn't. Paylod is the... lighter version of... payload... or something... 🤦‍♂️

Comment (kappa90) [reply ee/api/test/test_integration.py]: Ah lol

**Pull request merged**